android/src/main/kotlin/com/apptentive/apptentive_flutter/ApptentiveFlutterPlugin.kt

```diff
@@ -266,14 +283,13 @@ class ApptentiveFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware
     }
     val pushProvider: Int = parsePushProvider(pushProviderCode)
     try {
-      Apptentive.setPushNotificationIntegration(pushProvider, token)
+      Apptentive.setPushNotificationIntegration(application, pushProvider, token)
```

Comment:
Do we want/need a null check around application? Is it possible to be null?

Reply:
It is a `lateinit` property. Once the flutter engine is attached to process the `application` is available. It is good.